### PR TITLE
chore: make updates to align with autoprotocol-service

### PR DESCRIPTION
**Pull request by EribertoLopez**
Continuing updates for dataclasses and further typing...

### CODE REVIEW

File: autoprotocol/unit.py [hunk 1]
```diff
@@ -197,11 +197,19 @@ def __new__(cls, value, units=None):
...
-        if not units and isinstance(value, str):
-            try:
-                value, units = value.split(":")
-            except ValueError as e:
-                raise UnitStringError(value) from e
+        if not units:
+            if isinstance(value, str):
+                try:
+                    value, units = value.split(":")
+                    # value, units = Decimal(str(value)), units
```
Comment (joshhacksthings) [inline autoprotocol/unit.py:204]: is this a necessary comment?

**Review by joshhacksthings — DISMISSED** [review]
lgtm, just had a comment. also update changelog!

File: autoprotocol/unit.py (hunk 1 above)
Comment (EribertoLopez) [reply autoprotocol/unit.py:204]: nope, will remove

File: autoprotocol/builders.py [hunk 2]
```diff
@@ -27,13 +27,13 @@
...
 from collections import defaultdict
 from collections.abc import Iterable  # pylint: disable=no-name-in-module
-from dataclasses import dataclass
 from functools import reduce
 from numbers import Number
 from typing import Any, Dict, List, Optional, Union
 
 from .constants import SBS_FORMAT_SHAPES
 from .container import Container, Well, WellGroup
+from .types.protocol import *  # pylint: disable=unused-wildcard-import
```
Comment (valtriadi) [inline autoprotocol/builders.py:36]: Super nit (optional):  I understood that it was for shortened import statement to use wildcard, but I wonder if listing out all the actual items being imported would be more readable? 
My personal perspective on using wildcard was:
1) May have impact on the intialization processor of the module during runtime if the types/protocol.py become large in the near future. 
2) Less readable for troubleshooting/debugging the classes being used in this module - not immediately telling us where those files come from ...
3) May be risky of colliding namespace of same classes could exist on two or more different libraries

But this is just my thought :-)

**Review by valtriadi — APPROVED** [review]
LGTM 👍 
(just tiny nitty thought below)

File: docs/changelog.rst [hunk 3]
```diff
@@ -2,6 +2,9 @@
 Changelog
 =========
 
+* :release: `10.2.0 <2023-02-22>`
+* :support: `385` Propagating properties now adds instead of sets for acoustic spread
+
```
Comment (EribertoLopez) [inline docs/changelog.rst:7]: update changelog

File: autoprotocol/types/protocol.py [hunk 4]
```diff
@@ -190,11 +201,56 @@ class GelPurifyExtract:
...
+@dataclass
+class FlowCytometryChannel:
+    emission_filter: FlowCytometryChannelEmissionFilter
+    detector_gain: VOLTAGE
+    measurements: Optional[FlowCytometryChannelMeasurements] = None
+    trigger_threshold: Optional[int] = None
+    trigger_logic: Optional[FlowCytometryChannelTriggerLogic] = None
+
+
+@dataclass()
```
Comment (EribertoLopez) [inline autoprotocol/types/protocol.py:242]: nit: remove ()

**Review by valtriadi — APPROVED** [review]
👍 👍

**Pull request merged**